### PR TITLE
Update Trust WSes

### PR DESCRIPTION
**Pull request by helixhamin**
**_I affirm:_**
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x] that I agree to LandSandBoat's [Limited Contributor License Agreement](https://github.com/LandSandBoat/server/blob/base/.github/CONTRIBUTOR_AGREEMENT.md), as written on this date
- [x] that I have **read the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)**
- [ ] that I've _**tested my code and things my code changed**_ since the last commit in the PR, and will test after any later commits

Please note, I still need to test. I will change the "I have tested" to checked once I have tested.
Adds all Trust WSes that do not need scripts created (already exist).
Adds commented out trust WSes for those that still need scripts or mob_skill entry for easy reference when the skills can be scripted.

### CODE REVIEW

File: sql/mob_skill_lists.sql [hunk 1]
```diff
@@ -3531,73 +3571,195 @@ INSERT INTO `mob_skill_lists` VALUES ('TRUST_Maat',1048,1028); -- Shoulder Tackl
...
+-- INSERT INTO `mob_skill_lists` VALUES ('TRUST_Cid',1052,3322);            -- Critical Mass
+-- INSERT INTO `mob_skill_lists` VALUES ('TRUST_Cid',1052,3323);            -- Fiery Tailings
+INSERT INTO `mob_skill_lists` VALUES ('TRUST_Gilgamesh',1053,146);          -- Tatsun: Goten
+INSERT INTO `mob_skill_lists` VALUES ('TRUST_Gilgamesh',1053,152);          -- Tachi: Kasha
+-- INSERT INTO `mob_skill_lists` VALUES ('TRUST_Gilgamesh',1053,3434);      -- Tachi: Kamai
+-- INSERT INTO `mob_skill_lists` VALUES ('TRUST_Gilgamesh',1053,3435);      -- Iaikuki
+INSERT INTO `mob_skill_lists` VALUES ('TRUST_Areuhat',1054,37);             -- Seraph Blade
+INSERT INTO `mob_skill_lists` VALUES ('TRUST_Areuhat',1054,40);             -- Vorpal Blade
+INSERT INTO `mob_skill_lists` VALUES ('TRUST_Areuhat',1054,42);             -- Savage Blade
+INSERT INTO `mob_skill_lists` VALUES ('TRUST_Areuhat',1054,3438);           -- Dragon Breath
```
Comment (helixhamin) [inline sql/mob_skill_lists.sql:3595]: Wrong animation

File: sql/mob_skill_lists.sql [hunk 2]
```diff
@@ -3531,73 +3571,195 @@ INSERT INTO `mob_skill_lists` VALUES ('TRUST_Maat',1048,1028); -- Shoulder Tackl
...
+INSERT INTO `mob_skill_lists` VALUES ('TRUST_Areuhat',1054,3439);           -- Hurricane Wing
```
Comment (helixhamin) [inline sql/mob_skill_lists.sql:3596]: Wrong animation

File: sql/mob_skill_lists.sql [hunk 3]
```diff
@@ -3531,73 +3571,195 @@ INSERT INTO `mob_skill_lists` VALUES ('TRUST_Maat',1048,1028); -- Shoulder Tackl
...
--- INSERT INTO `mob_skill_lists` VALUES ('TRUST_Ferreous_Coffin',1059,0);
--- INSERT INTO `mob_skill_lists` VALUES ('TRUST_Lilisette',1060,0);
--- INSERT INTO `mob_skill_lists` VALUES ('TRUST_Mumor',1061,0);
--- INSERT INTO `mob_skill_lists` VALUES ('TRUST_Uka_Totlihn',1062,0);
--- INSERT INTO `mob_skill_lists` VALUES ('TRUST_Klara',1063,0);
--- INSERT INTO `mob_skill_lists` VALUES ('TRUST_Romaa_Mihgo',1064,0);
+INSERT INTO `mob_skill_lists` VALUES ('TRUST_Elivira',1056,209);            -- Split Shot
+INSERT INTO `mob_skill_lists` VALUES ('TRUST_Elivira',1056,212);            -- Slug Shot
+INSERT INTO `mob_skill_lists` VALUES ('TRUST_Elivira',1056,214);            -- Heavy Shot
+INSERT INTO `mob_skill_lists` VALUES ('TRUST_Elivira',1056,1201);           -- Coronach
```
Comment (helixhamin) [inline sql/mob_skill_lists.sql:3604]: Wrong animation

**Review by TeoTwawki — CHANGES_REQUESTED** [review]
use retail captures to verify the skill IDs (offset by 256) and animation IDs the trusts use. These will 99% of the time _not_ match the player type ones in the 0-255 range. Any link to the post in the captures discord or the capturelog that shows it would be very appreciated to save time.

File: sql/mob_skill_lists.sql [hunk 4]
```diff
@@ -3531,73 +3571,195 @@ INSERT INTO `mob_skill_lists` VALUES ('TRUST_Maat',1048,1028); -- Shoulder Tackl
...
+INSERT INTO `mob_skill_lists` VALUES ('TRUST_Noillurie',1057,150);          -- Tachi: Yukikaze
+INSERT INTO `mob_skill_lists` VALUES ('TRUST_Noillurie',1057,151);          -- Tachi: Gekko
+INSERT INTO `mob_skill_lists` VALUES ('TRUST_Noillurie',1057,152);          -- Tachi: Kasha
+INSERT INTO `mob_skill_lists` VALUES ('TRUST_Noillurie',1057,1197);         -- Tachi: Kaiten
```
Comment (helixhamin) [inline sql/mob_skill_lists.sql:3608]: Animation seems to be wrong, no swing with the weapon when he does ws

File: sql/mob_skill_lists.sql (hunk 3 above)
Comment (TeoTwawki) [reply sql/mob_skill_lists.sql:3604]: The entire table was prefilled with wrong IDs more than 6 years ago and everyone has been correcting them one at a time as they are implemented.

Really do not need a pr comment for every individual row for previously unimplemented skills that don't have a correct animation. just don't implement without getting the right one - that blows up everyone's inboxes so comment the row, not the PR.

Comment (TeoTwawki) [reply sql/mob_skill_lists.sql:3604]: p.s. allowing wrong animation IDs to activate on a running server can crash clients, so I really do not wish to see skills enabled with wrong IDs.

File: sql/mob_skill_lists.sql [hunk 5]
```diff
@@ -3531,73 +3571,195 @@ INSERT INTO `mob_skill_lists` VALUES ('TRUST_Maat',1048,1028); -- Shoulder Tackl
...
+INSERT INTO `mob_skill_lists` VALUES ('TRUST_Lhu_Mhakaracca',1058,68);      -- Spinning Axe
+INSERT INTO `mob_skill_lists` VALUES ('TRUST_Lhu_Mhakaracca',1058,69);      -- Rampage
+INSERT INTO `mob_skill_lists` VALUES ('TRUST_Lhu_Mhakaracca',1058,72);      -- Decimation
+INSERT INTO `mob_skill_lists` VALUES ('TRUST_Lhu_Mhakaracca',1058,1192);    -- Onslaught
```
Comment (helixhamin) [inline sql/mob_skill_lists.sql:3612]: Wrong animation

**Comment by TeoTwawki** [comment]
I mean it, stop making a new GitHub comment for every row. there will be THOUSANDS like this.

**Comment by helixhamin** [comment]
Sorry Teo, wasn't looking at this page. I will note it somewhere else.

**Pull request closed without merging**